tests/Tests/600__ai_apps/620__intel_aikit/620__intel_aikit.robot

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Resource        ../../../Resources/Page/LoginPage.robot
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource        ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource        ../../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+Library         SeleniumLibrary
+Suite Setup     Intel_Aikit Suite Setup
+Suite Teardown  Intel_Aikit Suite Teardown
+
+*** Variables ***
+${intel_aikit_appname}           aikit
+${intel_aikit_container_name}    Intel® oneAPI AI Analytics Toolkit Container
+${intel_aikit_operator_name}    Intel® oneAPI AI Analytics Toolkit Operator
+${image_path}                   image-registry.openshift-image-registry.svc:5000/redhat-ods-applications
+
+*** Test Cases ***
+Verify intel aikit Is Available In RHODS Dashboard Explore Page
+  [Tags]  ODS-1017 Smoke  Sanity
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
+  Verify Service Is Available In The Explore Page     ${intel_aikit_container_name} 
+  Verify Service Provides "Get Started" Button In The Explore Page     ${intel_aikit_container_name} 
+
+Install intelaikit From the Operatorhub
+   [Tags]  ODS-760   Sanity
+    Open Installed Operators Page
+    Search Installed Operator  ${intel_aikit_operator_name} 
+    ${status}    Is Operator Installed     ${intel_aikit_operator_name}
+    IF  not ${status}
+         Open OperatorHub
+         ${no_of_operator}    Get The Number of Operator Available    ${intel_aikit_appname}
+         IF  ${no_of_operator} == ${2}
```

Comment:
I'm not sure I understand this? Why are you counting how many operators are available? What happens if this number changes in the future?

Reply:
As of now we have two operator for aikit (marketplace and no marketplace each). Here i just wanted to make sure when we search there should be two operator. In case in future  the number gets changes we should be notified and make change. However IMHO it shouldn't change